docs/content/en/docs/Design/do_transformation.md

```diff
@@ -0,0 +1,179 @@
+<!-- mdformat off(yaml frontmatter) -->
+---
+title: Data-oblivious Transformations
+weight: 9
+---
+<!-- mdformat on -->
```

Comment:
```suggestion
---
title: Data-oblivious Transformations
weight: 9
---
```

Prompted by this PR, I have been tinkering with the auto-formatter in https://github.com/google/heir/pull/937. If you remove these html-style comments (as in that PR) then you should be able to run `mdformat --wrap=80` to get this PR to a consistent formatting style.